howto/android/custom-vhal.md

```diff
@@ -136,6 +136,100 @@ addons:
   - custom-vhal
 ```
 
+# Integrate Anbox VHAL interface
+
+To use and integrate the [Anbox VHAL interface](https://github.com/canonical/vendor_canonical_interfaces/tree/main/vehicle) into a VHAL implementation, the implementation must be built against Android 14. Before proceeding, ensure you have initialized the Android 14 manifest repository. If not, please follow the [official documentation](https://source.android.com/docs/setup/download) to set it up.
```

Comment:
The spellcheck will still flag initialized because the style switch to US English was just announced. Can we continue using initialised until all infrastructure is in place for the switch? Another option is to add to ignore list but that will keep getting longer.

```suggestion
To use and integrate the [Anbox VHAL interface](https://github.com/canonical/vendor_canonical_interfaces/tree/main/vehicle) into a VHAL implementation, the implementation must be built against Android 14. Before proceeding, ensure you have initialised the Android 14 manifest repository. If you do not yet have the manifest repository ready, follow the [official documentation](https://source.android.com/docs/setup/download) to set it up.
```

Reply:
+1.
FIxed it